about.html

```diff
@@ -0,0 +1,148 @@
+                <li><a href="#news">News</a></li>
+            </ul>
+            <button type="submit" class="campaign">Learn more</button>
+        </div>
+            
+        </nav>
+         <!-- head content  -->
+        <div class="head_content">
+            <h2>Find that <br>elusive Unicorns</h2>
+        </div>
+        <div class="header_quote">
+            <p>We at Sano have been in business since 2018, we have assist over millions of people find the right unicorn to invest into . We discover Unicorns with potential of becoming a successful startups and help them look for potential investors and also help investors by giving them the good investment opportunity.</p>
+        </div>
+        <div class="invitation">
+            <p>Please contact us through email for futher </p>
+            <p>questions about SANO</p>
+            <p class="underline"><strong>support@sano.org</strong></p>
+        </div>
+    </div>
+     <!-- about section  -->
+    <div class="logo" id>    
+        <div class="logo_content">
+            <h4>ABOUT SANO LOGO</h4>
+            <img src="./images/line2.png" alt="line" width="160px" height="20px">
+            <p class="logop"><strong>Lorem, ipsum dolor sit amet consectetur adipisicing elit. Soluta molestias nam similique numquam amet magni,<br> 
+                voluptates facere consequuntur<br> 
+                fugit aliquid illo incidunt quod accusantium
+                 laudantium nesciunt. Itaque, quis. Temporibus, 
+                 ad!
+            </strong></p>
+                 <img class="aboutlogo" src="./images/thelogo (1).png" alt="logo" width="250px" height="250px">
+        </div>    
+    </div>
+
+    <!-- past section  -->
+    <div class="past_content">
+
+        <ul class="gallery">
+            <li class="group">
+                <img src="./images/group1.jpg" alt="group1">
+              <div class="group-info">
+                <h2 class="past-year">2017</h2>
+                <p>SANO sumit in Lagos 2017</p>
+              </div>
+            </li>
+            <li class="group">
+                <img src="./images/group2.jpg" alt="group2">
+                <div class="group-info">
+                    <h2 class="past-year">2018</h2>
+                    <p>SANO sumit in Abuja 2018</p>
+                  </div>
+            </li>
+          </ul>
+
+    </div>
+ <!-- partner section  -->
+    <div class="partner">
+        <div class="partners">
+            <h2>Partners</h2>
+            <img src="./images/line2.png" alt="line" width="50px">
+        </div>
+            <div class="row1">
+                <h2><strong>mozilla</strong></h2>
+                <h3>Google</h3>
+                <h1><strong>NAVER</strong></h1>          
+            </div>
+            <div class="row2">
+                <h3>daumkakao</h3>
+                <h3><span><img src="./images/airbnb.png" alt="airbnb" width="36px" height="36px"></span>airbnb</h3>
+            </div>
+    </div>
+
```

Comment:
- [ ] The desktop view of the about section doesn't have an about section kindly consider removing this section from your page of the desktop view